CHANGELOG.md

```diff
@@ -14,9 +14,14 @@ the code was deployed.
 ### Added
 
 - CORS configuration to Express Proxy Middleware.
-- "doorMissedFrequently" field to JSON data submitted by brave sensors that is sent to /api/heartbeat (CU-860rk8v2a)
-- Sentry log in the case that doorMissedFrequently is true in posted data from brave sensor (CU-860rk8v2a)
+- Added a timer in state0 that is reset when the door closes where the sensor is allowed to detect for occupation (CU-8678t4ztg).
+- Added console function for the occupation detection timer as well as the timer count and timeout value in Debug Publishes (CU-8678t4ztg).
+- "doorMissedFrequently" field to JSON data submitted by brave sensors that is sent to /api/heartbeat (CU-860rk8v2a).
+- Sentry log in the case that doorMissedFrequently is true in posted data from brave sensor (CU-860rk8v2a).
 - GitHub Actions to deploy to AWS infrastructure on Dev and Staging (CU-860ra8f7q).
+### Fixed
+
+- Wording in consoleFunctionTests.cpp when functions are called with 'e'.
```

Comment:
I usually pull from main, fix merge conflicts and then recommit. Would you recommend that I rebase moving forward? From what I've read, rebasing recommits every one of my changes as if I had done it after everyone else's changes. The "merge" approach is what I used for this feature, but I will try rebasing for future commits if you think it's better.

Reply:
I find that sometimes merging into my branch creates a weird `diff` situation in GitHub where it shows code that I merged in as though I wrote it new in my branch. And that gets confusing. If you can avoid this, then merging is fine. @micahdbak also used merging in his most recent PR. I use rebase because it gives me a lot of control, but it does carry risk in that it's changing the past and needs a force push every time I use it.